packages/graphql-migrations/src/abstract/generateAbstractDatabase.ts

```diff
@@ -103,7 +103,7 @@ class AbstractDatabaseBuilder {
     for (const key of Object.keys(this.typeMap)) {
       const type = this.typeMap[key]
       // Tables
-      if (isObjectType(type) && !type.name.startsWith('__') && !ROOT_TYPES.includes(type.name)) {
+      if (isObjectType(type) && !type.name.startsWith('__') && !ROOT_TYPES.includes(type.name) && isModelType(type)) {
```

Comment:
hard-coded this for now, to prevent errors in migrations

Reply:
Actually missing context. Why migrations are affected by the filtering work? Is that we provide some extra types? Migration obviously ignores input types right ?

Reply:
We are now creating object types to wrap the list of models for returning. These do not have an `id: ID` field, but migrations expects this on all types.